style.css

```diff
@@ -68,23 +72,62 @@ nav li {
 .pageOne {
     height: 700px;
     opacity: 0.6;
+    background-image: url("tree.png");
+    background-size: 100% 100%;
 }
 
 .pages {
     height: 700px;
```

Comment:
Is 700px an arbitrary figure? if so, could it cooridinate with an actual screen-size?

Reply:
its a baseplate div for all main divs

Reply:
Yeah, but it doesn't fit my screen. So some actal screen sizing might be useful? if you want them to be screen sized shifts between pages. Obv you can't do this for all different screen sizes but maybe a bit bigger would cover more types of screen?